.github/workflows/nightly.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish Nightly Builds
+
+on:
+  push:
+    branches:
+      - nightly_build
+
+jobs:
+  build-test-publish:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - runson: macos-13 # mac x64
+            archive: async-profiler-3.0-macos-x64.zip
+
+          - runson: macos-14 # mac arm
+            archive: async-profiler-3.0-macos-arm64.zip
```

Comment:
Why have you changed it to build two separate binaries?
Previous async-profiler releases included just one muti-arch async-profiler package for macOS - I think this is convenient from UX perspective.

Reply:
I can change it back, should I only upload the build artifacts from the arm runner and only run the tests in x86?